notebooks/lesson12_nlp-classic.ipynb

```diff
@@ -0,0 +1,1416 @@
+{
```

Comment:
<p>it would be interesting to see how a random forest performs here as a nice empirical proof about the curse of dimensionality</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>